ddtrace/internal/ci_visibility/writer.py

```diff
@@ -29,6 +24,17 @@
 from .encoder import CIVisibilityEncoderV01
 
 
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import Dict
+    from typing import List
+    from typing import Optional
+
+    from ddtrace.vendor.dogstatsd import DogStatsd
+
+    from ...sampler import BasePrioritySampler
```

Comment:
This doesn't follow the usual pattern of importing type objects `if TYPE_CHECKING`. Why is this import conditional?

Reply:
Oh, I see! Thanks for clarifying that.